src/routes/tools/local-climate-change-snapshot/index.svelte

```diff
@@ -34,12 +34,82 @@
 </script>
 
 <script>
+  import { goto, stores as sapperStores } from "@sapper/app";
+  import { onMount } from "svelte";
+  import { getLocationFromQuery } from "./_helpers";
+  import { locationStore } from "~/routes/tools/_common/stores";
+  import { serialize } from "~/helpers/utilities";
   import { Resources } from "~/components/tools/Partials";
   import { Header } from "./_common";
   import SelectLocation from "./_select-location/SelectLocation.svelte";
 
   export let tool;
   export let resources;
+
+  const { page } = sapperStores();
+  const { location, boundary } = locationStore;
+
+  let selectedLocation = null;
+  let searchValue = "";
+  let selectedRadio = "locagrid";
+
+  onMount(() => {
+    cleanUpPrevDom();
+    initApp();
+  });
+
+  async function initApp() {
+    let loc;
+    let {
+      query: { lat, lng, boundary: boundaryType },
+    } = $page;
+    lat = +lat;
+    lng = +lng;
+    try {
+      loc = await getLocationFromQuery({
+        lng,
+        lat,
+        boundaryType,
+        fallbackLocation: undefined,
+      });
+    } catch (error) {
+      console.log(error);
+    }
+    if (loc) {
+      selectedLocation = loc;
+      searchValue = loc.title;
+      // TODO: override when fallback geom is present.
+      selectedRadio = boundaryType;
```

Comment:
Address this maybe by doing

```js
selectedRadio = selectedLocation === DEFAULT_LOCATION ? "locagrid" : boundaryType;
```
but `DEFAULT_LOCATION` has its geometry modified to be a point so might want to check against `searchValue` instead, e.g.

```js
selectedRadio = searchValue === DEFAULT_LOCATION.title ? "locagrid" : boundaryType;
```

Reply:
Actually this isn't necessary or desirable as we want the index page to load with no location selected if a valid one is not found via the URL query params.